packages/fxa-content-server/app/tests/spec/lib/router.js

```diff
@@ -514,72 +535,60 @@ describe('lib/router', () => {
 
   describe('React-related methods', () => {
     beforeEach(() => {
-      const mockAlwaysShownReactGroups = [
-        'simpleRoutes',
-        'resetPasswordRoutes',
-      ];
-      sinon
-        .stub(router, 'ALWAYS_SHOWN_REACT_GROUPS')
-        .value(mockAlwaysShownReactGroups);
+      sinon.spy(router, 'showReactApp');
     });
-    describe('showReactApp', () => {
-      beforeEach(() => {
-        sinon.spy(router, 'showReactApp');
-      });
 
+    describe('showReactApp', () => {
       describe('returns true', () => {
-        it('when routeName is included in react group route and user is in experiment', () => {
+        it('when feature flag is on, route is not fully rolled out in production, but user is in experiment group', () => {
           sinon.stub(router, 'isInReactExperiment').callsFake(() => true);
-          assert.isTrue(router.showReactApp('cannot_create_account'));
+          assert.isTrue(router.showReactApp('sometimesOnRoute'));
         });
 
-        it('when routeName is in ALWAYS_SHOWN_REACT_GROUPS and user is not in experiment', () => {
-          assert.isTrue(router.showReactApp('legal'));
+        it('when feature flag is on, full prod rollout is undefined, but user is in experiment group', () => {
+          sinon.stub(router, 'isInReactExperiment').callsFake(() => true);
+          assert.isTrue(router.showReactApp('fullProdUndefinedRoute'));
         });
 
-        it('when routeName is a simpleRoute, relier is OAuth, and user is in experiment', () => {
-          sinon.stub(router, 'isInReactExperiment').callsFake(() => true);
+        // enrolment in experiment is irrelevant if fully rolled out in production
```

Comment:
TIL: _"Both enrolment and enrollment are variants of the same word. Enrollment is the standard American spelling in the English language, while enrolment is for British English."_

/laughs in Canadian-British English